pliers/extractors/base.py

```diff
@@ -110,8 +110,13 @@ def to_df(self, timing=True, metadata=False, format='wide',
         # Ideally, Extractors should implement their own _to_df() class method
         # that produces a DataFrame in standardized format. Failing that, we
         # assume self._data is already array-like and can be wrapped in a DF.
+        index_cols = []
+
         if hasattr(self.extractor, '_to_df'):
             df = self.extractor._to_df(self, **to_df_kwargs)
+            for col in self.extractor._log_attributes:
```

Comment:
I could be remembering wrong, but isn't the dict in `_log_attributes` stored as a string in a *single* column in the default (wide) format? If that's the case, then I think we should respect that in long format and keep storing them all in one field as a string. If I'm wrong and they're already represented as individual fields, then this is fine.

Reply:
I just tried it out with  another extractor (`PredefinedDictionaryExtractor`), and it look like by default you don't get log attributes back in `to_df` at all. 

So this is really just to allow specific extractors to if they so wish, without screwing up the long format.

Reply:
I'd also be fine without this all together, but @rbroc wanted a way to pass back tokenizer results for BERT (which IMO are not actual "features") in `to_df`,